paradox-menu.el

```diff
@@ -721,17 +730,18 @@ Status:  _i_nstalled _a_vailable _d_ependency _b_uilt-in
   "Move to previous entry, which might not be the previous line.
 With prefix N, move to the N-th previous entry."
   (interactive "p")
-  (paradox-next-entry (- n))
+  (paradox-next-entry (- (prefix-numeric-value n)))
```

Comment:
Thanks for the help!

The only thing I don't understand is this line.
IIUC, `prefix-numeric-value` is using for converting `P` arguments into `p`, but we're already using `p` here.

Reply:
The function takes a single `&optional` argument. The `interactive` spec of `"p"` is guaranteed to set this argument to an integer when the function is called as an interactive command, but evaluating `(paradox-previous-entry)` from Lisp will not do this. In other words, the optional argument has to fall back to a default value if it is not supplied.

An alternative to `(prefix-numeric-value n)` would be `(or n 1)`. In my own code I prefer the latter because it is simpler, but the former conveys more meaning. It's a matter of style and completely up to you, but I would definitely recommend using one or the other.